test/org/zaproxy/zap/extension/openapi/HTTPDTestServer.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Partially this occurred to me this morning when I was thinking about: https://github.com/zaproxy/zaproxy/pull/3204 on the bus on the way to work. There are likely duplicate files for the Requester add-on that need to be similarly modified :( (Because they're duplicated from main to extension... ), eg: it has it's own copy of `HttpPanelSender.java`